src/modules/main/components/ImportantContainer.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Carousel from '../../../lib/components/Carousel';
+import { PostCard } from '../../../lib/components/PostCard';
+
+import { IRootState } from '../../../store/rootReducer';
+import { MOCK_DATA } from '../../../lib/constants/mock-data';
+import { loadImportant } from '../store/actions';
+
+const selectImportant = (state: IRootState) => state.main.important;
+
+const ImportantContainer: React.FC= () => {
+  const posts = useSelector(selectImportant);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    Promise.resolve(Array(8).fill(MOCK_DATA)).then((d) => {
+      dispatch(loadImportant(d));
+    })
+    .catch(console.log);
+  }, []);
```

Comment:
pls load the data using redux:
- data is not passed to the action
- data is fetched on the action call
- data is obtained on the component using the selector

Reply:
moved data fetching into `loadImportant`. maybe we should extract data fetching into separate function and pass it in action creators, so there is less repetition?

Reply:
might be a good idea though!
when we will use the real BE, we could create a service to fetch the data and use it in the action.